components/breadcrumb/stories/breadcrumb.stories.js

```diff
@@ -53,21 +53,252 @@ Default.args = {
 
 export const NestedMultiline = Template.bind({});
 NestedMultiline.args = {
+	items: [
+		{
+			iconName: "FolderOpen",
+		},
+		{
+			label: "Sub Item",
+		},
+		{
+			label: "Trend",
+			isDragged: true,
+		},
+		{
+			label: "January 2019 Assets",
+		},
+	],
+	variant: "multiline",
+};
+
+/*
+ * Stories for the MDX "Docs" only.
+ * Based off of the base `Template` which does not have conditional Chromatic-only markup.
+ */
+export const Dragged = Template.bind({});
+Dragged.args = {
 	items: [
 		{
 			label: "Nav root",
 		},
+		{
+			label: "Trend",
+			isDragged: true,
+		},
+		{
+			label: "January 2019 Assets",
+		},
+	],
+	isDragged: true,
+};
+Dragged.tags = ["docs-only"];
```

Comment:
Would you mind going through and updating with the new tags?

["docs-only"] == ["autodocs", "!dev"]

Reply:
Thanks for catching that: https://github.com/adobe/spectrum-css/pull/2793/commits/205960b4af7c2b6bfcef76347662f5d59b56a524